docker/start-op-geth.sh

```diff
@@ -46,7 +47,20 @@ then
     if [ ! -d $DATADIR ]
     then
         wget "https://raw.githubusercontent.com/base-org/node/main/goerli/genesis-l2.json" -O ./genesis-l2.json
-        exec geth init --datadir=$DATADIR ./genesis-l2.json
+        geth init --datadir=$DATADIR ./genesis-l2.json
+    fi
+elif [ $NETWORK = "custom" ] || [ $NETWORK = "devnet" ]
+then
+    if [ $NETWORK = "devnet" ]
+    then
+         rm -rf $DATADIR
```

Comment:
Why are we deleting the datadir if it is a devnet? Wouldn't this cause us to resync from genesis on every restart?

Reply:
The idea is that when you're working on the devnet and making changes to the code of the node, you'd likely prefer to start syncing from a clean environment. But i agree it might not always be necessary.